### ISSUE TITLE
user curated block galleries

### ISSUE THREAD

**Issue by enjalot** [enjalot/blockbuilder#95]
a logged in user could create gallery pages with some description and a collection of blocks. Adding blocks to the page would be done by pasting in any relevant url for a block (gist, block, blockbuilder, just the UUID etc.)
the gallery should display thumbnails similar to the user profile page on bl.ocks.org
the curator can reorder the thumbnails via drag and drop or by sorting on creation date.

**Comment by emeeks** [comment]
That's everything that I would like to see.

Here are some wants:
Ability for (logged in?) users to leave comments on the gallery.
Name the gallery
Provide Subsections
Default to "display mode" that hides the code

**Comment by micahstubbs** [comment]
this feature has gotten some interest on twitter today 
https://twitter.com/NadiehBremer/status/892463977934008321

**Comment by micahstubbs** [comment]
a nice example from @nbremer of what such a gallery could look like https://nl.pinterest.com/nadiehbremer/dataviz-d3js-blocks/

**Comment by micahstubbs** [comment]
related to ideas of tags, tagging, pins, pinning blocks

**Comment by nbremer** [comment]
Pinning felt like the best option for now :) But one issue I had was that not everybody had thumbnails, or had decent enough thumbnails (properly showing the project), so I had to make my own screenshots. But still, that was only for about 25%, the thumbnails on the other 75% saved me a lot of time :)

**Comment by micahstubbs** [comment]
good point!  it sounds like this has a soft dependency on #171 auto screenshot with gist-snap
